pages/blog/release-notes-2.1.0.md

```diff
@@ -0,0 +1,108 @@
+---
+title: AsyncAPI Spec 2.1.0 Release Notes
+date: 2021-06-29T06:00:00+01:00
+type: Communication
+tags:
+  - specification
+  - release
+cover: /img/posts/release-notes-2.1.0/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: 'The eagle has landed! Check out all the changes the AsyncAPI specification introduces in the new v2.1.0 release'
+featured: true
+---
+
+The last AsyncAPI release (2.0.0) took place on the 11th of September, 2019. In 2020 the focus went into growing community and adoption and stabilization of basic tooling for specification. This year was a year of "formalizm" 😃 aka getting into the foundation, setting up governance model and contribution guide to enable work on next spec release. We are good to go forward. The 2.1.0 release is out in the wild 🎉
+
+## Message examples object extended with additional fields
+
+Thanks to work done by [Laurent Broudoux](https://github.com/lbroudoux), you can now clearly describe message examples. New properties **name** and **summary** are optional. These properties help not only to properly describe the example in documentation but make it easier to work with mocking and testing tools (like [microcks](https://microcks.io/)), so you can better identify what example to use for mocking and what it does.
+
+Example of new properties added to existing [WebSocket example for Gemini API](https://github.com/asyncapi/spec/blob/v2.1.0/examples/websocket-gemini.yml):
+
+<CodeBlock highlightedLines={[10,11,25,26]}>
+{`components:
+  messages:
+    marketData:
+      summary: Message with marked data information.
+      description: |
+        The initial response message will show the existing state of the order book. Subsequent messages will show all executed trades, as well as all other changes to the order book from orders placed or canceled.
+      payload:
+        $ref: '#/components/schemas/market'
+      examples:
+        - name: updateMessage
+          summary: Example of an update message that contains a change in price information.
+          payload:
+            type: update
+            eventId: 36902233362
+            timestamp: 1619769673
+            timestampms: 1619769673527
+            socket_sequence: 661
+            events:
+              - type: change
+                side: bid
+                price: '54350.40'
+                remaining: '0.002'
+                delta: '0.002'
+                reason: place
+        - name: heartbeatMessage
+          summary: Example of additional heartbeat message when you enable them. 
+          payload:
+            type: heartbeat
+            socket_sequence: 1656`}
+</CodeBlock>
+
+Rendering of new example properties in React component and HTML template:
+
+<Figure
+  src="/img/posts/release-notes-2.1.0/ui.webp"
```

Comment:
Please update the example to the newest one :) 

![image](https://user-images.githubusercontent.com/20404945/123643226-75b2b380-d824-11eb-901a-117e561b071e.png)

I changed the color of name and summary to be more accessibility friendly. @fmvilas Thanks for suggestion!

Reply:
done